code/modules/projectiles/magazines/rifles.dm

```diff
@@ -123,6 +123,15 @@
 	default_ammo = /datum/ammo/bullet/rifle/ap
 
 //-------------------------------------------------------
+//MR4A. It's a Designated Marksmen Rifle, picture the Marine SCAR-H.
+/obj/item/ammo_magazine/rifle/m4radmr
+	name = "\improper M4RA Standard Magazine"
+	desc = "Made in the same 10x24 as the Mark One & HPR, the M4RA uses a distinct M252C round. This round has a higher velocity and pressure than the standard cartridge used by the M41, dealing more damage per round, at the expense of more recoil. Not as high quality as the A19, but it works."
```

Comment:
```suggestion
	desc = "Made in the same 10x24 as the M41 series & HPR, the M4RA uses a distinct M252C round. This round has a higher velocity and pressure than the standard cartridge used by the M41 series, dealing more damage per round, at the expense of more recoil. Not as high quality as the A19, but it works."
```
directly quoted:

> In addition to this, while not directly stated in game, the weapon is chambered in M252 High Explosive Armor Piercing ammunition with a 15 round magazine; the same rounds as the aforementioned "M42A" (as one of its caliber options) and the M56 Smartgun. This can be seen in the markings below the ejection port of the weapon: "ARMAT M4RA" with "10mm M252 HEAP" below.